### DISCUSSION TITLE
Alignment of floating callout boxes

### DISCUSSION THREAD

**Discussion by Tanjamuse** [efemkay/obsidian-modular-css-layout#68] [Q&A] [answered]
How do I get the floating callout box aligned on the same line as the text to the left?

This is the css for my callout box:

```
.callout[data-callout="back"] {
	--callout-color: 255, 0, 0;
	--callout-icon: lucide-undo-2;
	border: 2px solid rgba(255, 0, 0, 0.90);
	border-radius: 20px;
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 5px;
	width: fit-content;
	cursor: pointer;
	font-family: "Sitka Text";
	font-size: 1em;
}
```

![Floating Callout](https://github.com/efemkay/obsidian-modular-css-layout/assets/15678057/1db44d05-00f8-46d2-b516-8a47884cd51c)

**Comment by efemkay** [comment, marked as answer]
sorry for the super late response. anyway, it doesn't matter the type of callout you use as MCL float uses the modifier. you can adjust the top alignment via style settings.

![image](https://github.com/efemkay/obsidian-modular-css-layout/assets/42369515/bc3eead4-ff7b-4b82-8287-65c26aa100e4)

**Comment by Tanjamuse** [comment]
Thank you so much 🧡  I hadn't realized you'd updated the css-files until now.
